TestingLowerBounds/ForMathlib/Integrable_of_empty.lean

```diff
@@ -0,0 +1,27 @@
+import Mathlib.MeasureTheory.Integral.Bochner
+
+open MeasureTheory
+
+open Real MeasureTheory Filter MeasurableSpace
+open scoped ENNReal NNReal Topology BigOperators
+
+--TODO: put this in mathlib, maybe just after:
+#check Integrable.of_finite
```

Comment:
Please remove the two `#check` commands: it makes the build step of CI noisy and makes it harder to see actual warnings and errors.